docs/themes/thxvscode/layouts/partials/breadcrumb.html

```diff
@@ -0,0 +1,59 @@
+{{/* 
+  Purpose: To generate breadcrumb navigation for documentation pages. The script processes the current page's URL to extract package names and 
+           section identifiers, mapping them to human-readable names. It constructs a hierarchical breadcrumb trail, indicating the current page's 
+           position within the site's documentation structure.
+  Inputs: The current page's URL and site data (e.g., package names, API details).
+  Output: An HTML paragraph (<p>) containing the breadcrumb links.
+*/}}
+
+{{ $concept := "Packages" }}
+{{ $leafSegment := "" }}
+{{ $leafNodeWithCapitalization := "" }}
+{{ $categoryCapitalized := "" }}
+{{ $leafNodeFound := false }}
+
+{{ $fullURL := trim .Page.RelPermalink "/" }}
+{{ $urlSegments := split $fullURL "/" }}
+{{ $packageName := index $urlSegments 2 }}
+
+{{ if $packageName }}
+  {{ $packageDisplayName := index $.Site.Data.displayNameToPackageName $packageName }}
+
+  {{ range $key, $value := $.Site.Data.packages }}
+    {{ if in $value $packageDisplayName }}
+      {{ $concept = $key }}
+      {{ break }}
+    {{ end }}
+  {{ end }}
+
+  {{ if gt (len $urlSegments) 3 }}
+    {{ $leafSegment = index $urlSegments 3 }}
+    {{ $leafNodeFound = true }}
+    {{ $leafParts := split $leafSegment "-" }}
+    {{ if eq (len $leafParts) 2 }}
+      {{ $leafNodeLower := index $leafParts 0 }}
+      {{ $categoryLower := index $leafParts 1 }}
+      {{ $categoryCapitalized = (title (humanize $categoryLower)) }}
+      {{ $allLeafNodes := index $.Site.Data.allAPIs $packageName }}
+      {{ $leafNodesInCategory := index $allLeafNodes $categoryCapitalized }}
+      {{ range $leafNodesInCategory }}
+        {{ if eq (lower .) $leafNodeLower }}
+          {{ $leafNodeWithCapitalization = . }}
+          {{ break }}
+        {{ end }}
+      {{ end }}
+    {{ end }}
+  {{ end }}
+
+  <p>
+    <a href="/docs/apis/" onclick="return false;">{{ humanize $concept | title }}</a> <!-- TODO: remove onclick and create proper link to new concept readme -->
```

Comment:
Is this TODO still relevant?

Reply:
Yes it is, Harshal is thinking about where that link should point to. It might be a new feature pointing to a content rich page explaining what the concept is